### PR TITLE
PHP 7.1, coding style

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

**Comment by mabar** [comment]
@f3l1x 
Tests could be fixed with `$now = strtotime((new DateTime('2014-1-2 00:00:00'))->__toString());` But it does not look like the right solution.

https://github.com/contributte/utils/blob/php7.1/tests/cases/DateTime.compare.phpt#L15
https://travis-ci.org/contributte/utils/jobs/407265721

**Comment by f3l1x** [comment]
`strtotime((string) ...)` is not working?

File: .travis.yml [hunk 1]
```diff
@@ -19,42 +18,41 @@ script:
...
 
     - stage: Test Coverage
-      php: 7.1
+      php: 7.2
       script:
         - composer run-script coverage
       after_script:
-        - wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar
+        - wget https://github.com/php-coveralls/php-coveralls/releases/download/v2.0.0/php-coveralls.phar
         - php coveralls.phar --verbose --config tests/.coveralls.yml
```
Comment (f3l1x) [inline .travis.yml]: Ping

**Review by f3l1x — CHANGES_REQUESTED** [review]

**Comment by mabar** [comment]
You are right. I just expected that __toString will be called automatically when required.

**Pull request merged**